### PR TITLE
Add Elixir 1.16 to pipeline

### PR DESCRIPTION
**Pull request by filmor**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -85,7 +85,8 @@ jobs:
     strategy:
       matrix:
         pair:
-          - { erlang: "26", elixir: "1.15", latest: true }
+          - { erlang: "26", elixir: "1.16", latest: true }
+          - { erlang: "26", elixir: "1.15" }
           - { erlang: "26", elixir: "1.14" }
           - { erlang: "25", elixir: "1.14" }
           - { erlang: "24", elixir: "1.13" }
```
Comment (evnu) [inline .github/workflows/main.yml:92]: Should we drop `1.13` with that? At least our current policy says we support the three newest OTP versions and the three newest Elixir versions.

Comment (filmor) [reply .github/workflows/main.yml:92]: I just didn't want to bother checking whether 1.14 supports OTP24 (which we still have to check). But yeah, we should probably adjust it.

Comment (evnu) [reply .github/workflows/main.yml:92]: According to [the docs](https://hexdocs.pm/elixir/1.16.1/compatibility-and-deprecations.html), Elixir 1.14 supports OTP 23-25 (and OTP 26 as of 1.14.5):

image

Comment (filmor) [reply .github/workflows/main.yml:92]: How about 24+1.14, 25+1.15, 26+1.15 and 26+1.16?

**Pull request merged**